src/common/paths/buckets_bucketID.yml

```diff
@@ -35,7 +35,7 @@ patch:
     content:
       application/json:
         schema:
-          $ref: "../schemas/Bucket.yml"
+          $ref: "../schemas/PatchBucketRequest.yml"
```

Comment:
Pinning the commit sha feels like a good move regardless (it's what I've been doing in the CLI). Hopefully if this does break anything it's seen as a "good" breakage since it's fixing an API mismatch, but feel free to point the UI team my way if they feel otherwise 😅 

Going to merge this for now to unblock @stuartcarnie